package.json

```diff
@@ -7,15 +7,13 @@
     "url": "https://github.com/moroshko/shallow-equal.git"
   },
   "author": "Misha Moroshko <michael.moroshko@gmail.com>",
+  "main": "src/index.js",
```

Comment:
do we also need a module key?

Reply:
I will add esm in the next PR. I like atomic pull requests.